### DISCUSSION TITLE
Installing radeon pro problem

### DISCUSSION THREAD

**Discussion by razvanrsk** [GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon#508] [Q&A]
Hello!

I downloaded la latest version of radeon pro render, i use 3.1.2 blender version  and when i want to install the script i get this error
"No module named __rpr"
![Capture](https://user-images.githubusercontent.com/71674203/168463538-fd567b99-0f03-4e28-b2ea-bf13923c1952.PNG)

**Comment by bar971** [comment]
I have same error... can be that we are using a steam installation?

**Reply by LebovskiOG** [reply]
I downloaded an old version of blender (2.80) and managed to install RadeonProRender on this version

**Comment by TAJ68PR** [comment]
Same issue I'm not using a steam installation.

**Reply by LebovskiOG** [reply]
I downloaded an old version of blender (2.80) and managed to install RadeonProRender on this version
